spec/integration/aws_auth_request_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'lite_spec_helper'
+require 'net/http'
+
+describe Mongo::Auth::Aws::Request do
+  require_aws_auth
+
+  let(:access_key_id) { ENV.fetch('MONGO_RUBY_DRIVER_AWS_AUTH_ACCESS_KEY_ID') }
+  let(:secret_access_key) { ENV.fetch('MONGO_RUBY_DRIVER_AWS_AUTH_SECRET_ACCESS_KEY') }
+  let(:session_token) { ENV['MONGO_RUBY_DRIVER_AWS_AUTH_SESSION_TOKEN'] }
```

Comment:
Should we put these in the spec config like the FLE environment variables?

Reply:
There are quite a few more env vars going with the aws auth testing, let's revisit this question once all of the tests land?

Reply:
Sure thing.